src/app/package-json.js

```diff
@@ -168,13 +168,6 @@ const packageJSON = (current, context) => {
     'author': context.author,
     'license': context.licenseName,
 
-    'browserify': {
-      transform: [
-        'browserify-shim',
-        'browserify-versionify'
```

Comment:
Hm... I'm actually not sure how it handles it. Would be good to try it out.
Also, we'll want to make sure that `browserify-versionify` is set as a dependency and not a dev dependency.